extensions/sequelize/src/sequelize/sequelize.repository.base.ts

```diff
@@ -975,6 +979,32 @@ export class SequelizeCrudRepository<
         } as typeof columnOptions);
       }
 
+      // For dialects that do not support a native JSON type, we need to parse
+      // string responses to JSON objects to preserve backwards compatibility with the Juggler ORM.
+      // Example: https://github.com/loopbackio/loopback-connector-mysql/blob/edf176b09234b82796f925203ff006843e045498/lib/mysql.js#L476
+      // With Sequelize v6, some of the dialects like MariaDB and SQLite already parse JSON strings to JSON objects by default:
+      //  - https://github.com/sequelize/sequelize/blob/cb8ea88c9aa37b14c908fd34dff1afc603de2ea7/src/dialects/mariadb/query.js#L191
+      //  - https://github.com/sequelize/sequelize/blob/cb8ea88c9aa37b14c908fd34dff1afc603de2ea7/src/dialects/sqlite/query.js#L365
+      if (
```

Comment:
Rather than checking for dialect, can we not have a flag that tells whether to do JSon stringification or not. That ways, for any DB other than postgres and mysql, it can be catered as well.

Reply:
@samarpanB Sure, how about a new option on the datasource for this? I set it to default to "true" for MySQL: [`4891c15` (#10285)](https://github.com/loopbackio/loopback-next/pull/10285/commits/4891c1504360aa67d5c5a0a9453b2dfd466968a1)